Models/RegistrationBase.php

```diff
@@ -359,12 +363,108 @@ public static function create($request)
         $reg->fill($inputs);
         $reg->save();
 
+
         $response['success'] = true;
         $response['data']['item'] = $reg;
         $response['messages'][] = trans('vaahcms-general.saved_successfully');
         return $response;
 
     }
+
+    //-------------------------------------------------
+
+    public static function createRegistration($request)
+    {
+
+        $inputs = $request->all();
+
+           $rules = [
+                'first_name' => 'required|max:150',
+                'last_name' => 'required|max:150',
+                'username' => 'required|max:150',
+               'email' => 'required|regex:/(.+)@(.+)\.(.+)/i',
+                'password' => [
+                    'required',
+                ],
+                'confirm_password' => 'required|same:password',
+            ];
+
+            $messages = [
+                'confirm_password.same' => 'Password and confirm password does not match.',
+            ];
+
+            $validator = \Validator::make($request->all(), $rules, $messages);
+
+        if ( $validator->fails() ) {
+            $errors = errorsToArray($validator->errors());
+            $response['success'] = false;
+            $response['errors'] = $errors;
+            return $response;
+        }
+
+        // check if already exist
+        $user = static::where('email',$inputs['email'])->first();
```

Comment:
@we-anil-y001 take same example
```
in DB: example@domain.com
in your query at line 406, if a user write Example@doMain.com
```
will your code work?

Reply:
It will not insert data if write this Example@doMain.com and this example@domain.com both are same email.com
check this video sir
https://img-v5.getdemo.dev/screenshot/chrome_m6FbYwNNbb.mp4